diagnostics-app/src/app/layout/header/header.component.ts

```diff
@@ -17,12 +17,18 @@ import { ThemeService } from 'src/app/core/services/theme.service';
   styleUrls: ['./header.component.css'],
 })
 export class HeaderComponent {
+  @Output() toggleDrawer = new EventEmitter<void>();
+
   constructor(private _themeService: ThemeService) {}
 
   onToggleTheme() {
     this._themeService.toggleTheme();
   }
 
+  onToggleDrawer() {
+    this.toggleDrawer.emit();
+  }
```

Comment:
we need to add tests for this method.

I am aware you are adding an integ. test in content-layout-component. But still, we need to test the behavior here.

Reply:
oh, you already added a unit-test, that's fine then.